ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Operating system
+
+*Which operating system are you working on? E.g., Ubuntu Linux, Max OSX, Windows 10, etc.*
+
+## Client version
+*Output of:*
+```
+import synapseclient
+synapseclient.__version__
+```
+
+## Description of the problem
+
+*Provide a description of the problem, and if possible a minimal reproducible example.*
```

Comment:
More information on what a minimal reproducible example is. Might be things like don't share private data with us.
I hesitate to put 'don't include extra imports, etc.' because many people may stop from filing if they have to do too much work, but I could be convinced.

Reply:
I think in some cases, the problem happens when using the package with other libraries. In those cases, we can't repro without extra imports. So I mentions "often do not requires extra imports."